edx_proctoring/static/proctoring/templates/course_allowances.underscore

```diff
@@ -0,0 +1,58 @@
+<span class="tip"> <%- gettext("Add Allowance for User: ") %>
```

Comment:
@chrisndodge Would like to avoid using "User" in the UI. Also, is this string necessary? 
Workflow seems fine without it: Click "+ Add Allowance" then fill out fields including "Username or Email".